AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -34,6 +34,11 @@ reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPa
 !
 mobile.donanimhaber.com##.mobil-forum-tanitim
 mobilkulup.com##.TapatalkBanner_Container
+ifeng.com###root > section[class^="fixSlide-"]
+ifeng.com###root > div[class^="bottom_box-"]
```

Comment:
this is description for photos
![image](https://user-images.githubusercontent.com/8361299/75985338-d104b800-5ef4-11ea-970e-8d2b965a20a5.png)

Reply:
but it doesn't lays in `#root` directly

Reply:
<details>
<summary>Screenshot </summary>
<img src="https://user-images.githubusercontent.com/18536328/76061643-06f97900-5f95-11ea-821f-535398b35d8e.png" />
</details>